src/core/blockchain/ethereum/account.ts

```diff
@@ -45,3 +57,11 @@ export const amountToStd = (value: BigNumber | number | string): BigNumber => {
 export const amountFromStd = (value: BigNumber): BigNumber => {
     return convertUnit(value, config.defaultUnit, config.coin, config);
 };
+
+export const convertToGasPriceUnit = (value: BigNumber): BigNumber => {
```

Comment:
we need to do this in a abstract way, there will be blockchains that doesn't have gas. However you can do a generic function `convertUnit()` and use it